install/configuring_inventory_file.adoc

```diff
@@ -852,6 +852,9 @@ Use the Device Mapper Storage Driver guide].
 available either locally or in at least one of the configured container image
 registries on the host machine.
 
+|`openshift_release`
+|Specifies the generic release of {product-title} for containerized installations. For RPM installations, refer to `package_availability`.
```

Comment:
Do you mean that for RPM installations you need to use `package_availability` instead? If so, I'd do something like s/refer to \`package_availability\`/set a \`package_availability\` value   (two places)

Reply:
I agree, thank you @kalexand-rh !